README.md

```diff
@@ -1 +1,15 @@
 # razor-vanagon
+
+## How to promote to PE
+
+1. Tag razor-server with the correct version (using Kerminator).
+1. Pin razor-vanagon to that version in configs/components/razor-server.json. Commit this to master.
+1. Tag razor-vanagon with the SHA of the commit above (using Kerminator).
+
+Git automation will generate pe-razor-server packages, then promote those packages into the latest version of PE.
```

Comment:
Did you set up automation to do this? because I'm not aware of any webhooks or anything set up to build pe-razor-server automatically.

Reply:
Isn't that the "razor-suite (master) PE Suite Pipeline Step 06 -- Vanagon Project Integration" step here: https://cinext-jenkinsmaster-enterprise-prod-1.delivery.puppetlabs.net/view/Razor/

I'm unclear on how releases happen for FOSS, i.e. how we add to GitHub's Releases page. I'd like that to be in this doc as well.

Reply:
ahh okay, yeah, looks like https://cinext-jenkinsmaster-enterprise-prod-1.delivery.puppetlabs.net/view/Razor/job/platform_razor-suite_razor-vanagon-suite-init_pe_suite_master/ (the most upstream job) does poll. I just didn't know that was set up. 😊 
so yeah, this is accurate.

The FOSS release process is in winston (https://github.com/puppetlabs/winston/blob/master/files/razor/server/release_ticket_descriptions.yaml) but looks like it's a bit out of date. Do you want those instructions in this doc as well or would it be better to just add instructions for generating winston tickets and just keeping the process up-to-date in that one place?

Reply:
I'm happy to link directly to that in this doc, so it only lives in one place.